modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheOperationContext.java

```diff
@@ -31,10 +32,17 @@
  * Cache operation context.
  */
 public class CacheOperationContext implements Serializable {
-    /** */
-    //TODO IGNITE-8801 remove this and set default as `false`.
-    public static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX =
```

Comment:
Let's keep this variable:
``
private static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX = false;

public static final boolean defaultAllowAtomicOpsInTx() {
     return IgniteSystemProperties.getBoolean(IGNITE_ALLOW_ATOMIC_OPS_IN_TX, DFLT_ALLOW_ATOMIC_OPS_IN_TX);
}

```

Reply:
thank you